lasagne/layers/base.py

```diff
@@ -300,3 +307,50 @@ def get_output_for(self, inputs, **kwargs):
         `NotImplementedError`.
         """
         raise NotImplementedError
+
+
+class UnLayer(MergeLayer):
+
+    """
+    lasagne.layers.UnLayer(incoming, layer, **kwargs)
```

Comment:
You can omit the signature here. It's only needed where the automatically generated signature doesn't look good because it contains objects (such as weight initializers). Also please remove the empty line before the docstring, just to be consistent with what we have elsewhere.